molecule/scenario.py

```diff
@@ -197,7 +197,8 @@ def idempotence_sequence(self):
 
     @property
     def lint_sequence(self):
-        return ['lint']
+        # see https://github.com/ansible/molecule/issues/2216
+        return ['dependency', 'lint']
```

Comment:
So now dependency is implicit in lint ...

Reply:
That is matching behavior with the other commands, run `molecule --help` and you will understand.